src/Admin/FieldDescription.php

```diff
@@ -15,6 +15,9 @@
 
 use Sonata\AdminBundle\Admin\BaseFieldDescription;
 
+/**
+ * @final since sonata-project/doctrine-orm-admin-bundle 3.x
```

Comment:
IMO we should ensure that all final we want to add is not extended anywhere in any of the sonata bundles. We have some cases where we marked something final for Next major and it cant happen because is extended somewhere else. Have you check all of them?

Reply:
AFAIK we do not extend any class from the doctrine specific admin bundles.